src/Model/Contact.php

```diff
@@ -727,6 +727,7 @@ public static function updateSelfFromUserID($uid, $update_avatar = false)
 		$fields['avatar'] = System::baseUrl() . '/photo/profile/' .$uid . '.' . $file_suffix;
 		$fields['forum'] = $user['page-flags'] == User::PAGE_FLAGS_COMMUNITY;
 		$fields['prv'] = $user['page-flags'] == User::PAGE_FLAGS_PRVGROUP;
+		$fields['unsearchable'] = $user['hidewall'] || $profile['net-publish'];
```

Comment:
Shouldn't it be `!$profile['net-publish']` instead? See `src/Model/GContact.php:1040`,

Reply:
You're welcome, I'm glad I caught it.